test/e2e/framework/log.go

```diff
@@ -49,6 +49,7 @@ func Failf(format string, args ...interface{}) {
 	skip := 2
 	log("FAIL", "%s\n\nFull Stack Trace\n%s", msg, PrunedStack(skip))
 	e2eginkgowrapper.Fail(nowStamp()+": "+msg, skip)
+	panic("unreachable")
```

Comment:
is it worth using static check pragmas here instead of panicing?
e.g. ` //nolint:staticcheck`

Reply:
I did start by adding the `//nolint:staticcheck` in several places but there ended up being many occurrences of SA5011 (I kept fixing a few and then more would show up the next time I linted; maybe the output was truncated) so I decided that adding the panic would be simpler. Adding the `panic("unreachable")` also appeared to be a recommended approach, from the docs that I linked in the PR description up top, for handling a case where staticcheck doesn't understand that the control flow has been aborted.